.travis.yml

```diff
@@ -15,6 +15,7 @@ android:
 
     # The BuildTools version used by your project
     - build-tools-26.0.1
```

Comment:
Let's remove this build tool. Possible?

Reply:
yes

Reply:
Great! Look forward to seeing the removal.